password-hash/src/params.rs

```diff
@@ -39,14 +39,14 @@ const MAX_LENGTH: usize = 8;
 ///
 /// [1]: https://github.com/P-H-C/phc-string-format/blob/master/phc-sf-spec.md#specification
 #[derive(Clone, Default, Eq, PartialEq)]
-pub struct Params<'a> {
+pub struct ParamsBuf<'a> {
```

Comment:
Note: this name change is motivated both by having a different name versus `T::Params` and some additional refactoring I'd like to do.